README.md

```diff
@@ -1,38 +1,53 @@
-# Dolphin memory engine
-A RAM search specifically made to search, track and edit at runtime the emulated memory of the Dolphin emulator (a Nintendo Wii and Nintendo GameCube emulator).  Its primary goal is to make research, TASing and reverse engineering of games more convenient and easier than the alternative solution, Cheat Engine.  Its name is derived from Cheat Engine to symbolize that.
+# Dolphin Memory Engine
+A RAM search specifically made to search, track, and edit the emulated memory of [the Dolphin emulator](https://github.com/dolphin-emu/dolphin) at runtime. The primary goal is to make research, tool-assisted speedruns, and reverse engineering of GCN & Wii games more convenient and easier than with the alternative solution, Cheat Engine. Its name is derived from Cheat Engine to symbolize that.
```

Comment:
I would keep the spelling of the console to "GameCube" and "Wii" because one: it's used so further and two: I am fine with abbreviations in this, but as long as they were defined before and for such a short README, I don't think it's necessarly worth to do that anyway.

Reply:
Yeah, that initialism was completely unnecessary. Fixed with dc0e1c7.

Sorry about the delay; it's been one hell of a day for me. 😖